### PR TITLE
Update lab-example clos02

### PR DESCRIPTION
**Pull request by ubaumann**
To be able to configure the lab, I needed to add an address family in BGP and to bring the IP layer up, I had to add the "admin-state".

I also needed to install `gnmic` and change the password in the script.

This is my first time using Nokia SR Linux, so I hope I did everything correctly. I am happy to discuss the changes and update it.

### CODE REVIEW

File: lab-examples/clos02/setup.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 CFG_DIR=./configs
 
 configure_SRL() {
-  OUT=$(gnmic -a clab-clos02-$1 --timeout 30s -u admin -p admin -e json_ietf --skip-verify set --update-path / --update-file $CFG_DIR/$1.yaml 2>&1)
+  OUT=$(gnmic -a clab-clos02-$1 --timeout 30s -u admin -p $PASSWORD -e json_ietf --skip-verify set --update-path / --update-file $CFG_DIR/$1.yaml 2>&1)
```
Comment (hellt) [inline lab-examples/clos02/setup.sh:6]: it is fine to use the documented current password -- `NokiaSrl1!`

**Comment by hellt** [comment]
Thanks @ubaumann, this is a great contribution.
The reason you had to make those changes is because we didn't pin the SR Linux container to a particular version these initial config snippets were tested against.

Would you be able to ping the version in the clab file to `23.3.3` (current latest) so that others won't have the same issues should they appear with later releases?
Thanks!

File: lab-examples/clos02/setup.sh (hunk 1 above)
Comment (ubaumann) [reply lab-examples/clos02/setup.sh]: okay I changed it to the documented password. Was not sure if you want to have it in git or not

Comment (steiler) [reply lab-examples/clos02/setup.sh]: Maybe define the PASSWORD variable at the top.?!

Comment (ubaumann) [reply lab-examples/clos02/setup.sh]: Good idea. I hope it looks good now.

**Comment by ubaumann** [comment]
@hellt @steiler Hope it looks good now. If I can change anything else, please let me know.

**Pull request merged**